src/ServiceManagement/HDInsight/Commands.HDInsight/Cmdlet/NewAzureHDInsightClusterCmdlet.cs

```diff
@@ -47,6 +47,49 @@ public NewAzureHDInsightClusterCmdlet()
             this.command = ServiceLocator.Instance.Locate<IAzureHDInsightCommandFactory>().CreateCreate();
         }
 
+        /*
+         * Parameter Sets:
+         * 
+         * ParameterSetClusterByConfigWithSpecificSubscriptionCredentials
+         *      Position 00: Name
+         *      Position 01: Config
+         *      Position 02: Subscription
+         *      Position 03: Certificate
+         *      Position 04: Location
+         *      Position 05: Credential
+         *      Position 06: ???
```

Comment:
What does "???" mean? Please add comment if this position is skipped intentionally/by mistake.